internal/services/provisionner.go

```diff
@@ -149,6 +150,43 @@ func GenerateUserRoleBinding(namespace string, role string) {
 	clientSet, err := kubernetes.NewForConfig(kconfig)
 	api := clientSet.RbacV1()
 
+	roleBinding(role, api, namespace, subjectAdmin(namespace, role), err)
+	roleBinding("view", api, namespace, subjectView(namespace), err)
+}
+
+func subjectView(namespace string) []v1.Subject {
+	subjectView := []v1.Subject{
+		{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "Group",
+			Name:     fmt.Sprintf("%s-%s", namespace, utils.ApplicationViewer),
+		},
+	}
+	return subjectView
+}
+
+func subjectAdmin(namespace string, role string) []v1.Subject {
+	subjectAdmin := []v1.Subject{
+		{
```

Comment:
isn;t that exactly what subject view is doing, should role be passed as read only?
Wouldn't that deserve a proper refactor?